doc/sphinxext/cited_mne.py

```diff
@@ -37,12 +44,12 @@
       'Gecko/20100913 Firefox/3.6.9')
 
 # ##### Templates for citations #####
-html = (u""".. _cited
+html = (u""".. _cited:
 
 Publications by users
 =====================
 
-Papers citing MNE as extracted from Google Scholar (on %s).
+Papers citing MNE (%d) as extracted from Google Scholar (on %s).
```

Comment:
okay as you like :)

By the way, I noticed that cloning mne is taking much longer than it used to before. The repo size shown is 88 MB. I thought it was much smaller, no?